src/bun.zig

```diff
@@ -2091,8 +2091,46 @@ pub const Stat = if (Environment.isWindows) windows.libuv.uv_stat_t else std.os.
 
 pub var argv: [][:0]const u8 = &[_][:0]const u8{};
 
+extern "c" fn CommandLineToArgvW([*:0]u16, *c_int) ?[*][*:0]u16;
```

Comment:
This should go in `windows.zig`

Reply:
also it should use the winapi calling convention
```suggestion
extern "kernel32" fn CommandLineToArgvW([*:0]u16, *c_int) callconv(windows.WINAPI) ?[*][*:0]u16;
```